test-runner-frontend/src/Components/DropdownMenuAllTestRuns.js

```diff
@@ -0,0 +1,108 @@
+import * as React from 'react';
+import { styled, alpha } from '@mui/material/styles';
+import Button from '@mui/material/Button';
+import Menu from '@mui/material/Menu';
```

Comment:
looking at the figma designs, i think we actually want to be using a select component - https://mui.com/material-ui/react-select/. swapping to this should also help with us not needing to maintain whether it is open or not